CalendarApp/BookingRoom/Helpers/SettingsHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingRoom.Helpers
+{
+    public static class SettingsHelper
+    {
+        public static string GetLoggerName()
```

Comment:
I don't see any benefit having dedicated method or pair of methods per each app setting. 
Main goal of having this helper class is to avoid potential mistakes in appsetting names. My suggestion is to create pair of methods : one for getting app setting second for setting. They should accept value from enumeration that will be used to identified appsetting that needs to be received/updated